ldms/src/sampler/lustre_client/Plugin_lustre_client.man

```diff
@@ -33,13 +34,18 @@ name=<plugin_name>
 .br
 This MUST be lustre_client.
 .TP
-job_set=<metric set name>
+job_set=<job metric set name>
 .br
 The name of the metric set that contains the job id information (default=job_id)
+.TP
+producer=<alternate host name>
+.br
+The default used for producer (if not provided) is the result of gethostname().
+The set instance names will be $producer/$llite_name.
 .RE
 
-.SH BUGS
-No known bugs.
+.SH NOTES
+Improperly spelled option names are not trapped as configuration errors.
```

Comment:
I.E., just like all of the other ldms plugins. :) It would be great to improve the plugin API so every plugin author didn't need to repeat those checks over and over again.